packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -32,11 +32,6 @@ $input-label-weight: 400 !default;
 /// @group global-themes
 $disabled: $disabled-02 !default;
 
-/// @type Color
-/// @access public
-/// @group global-themes
-$disabled-background-color: $disabled-01 !default;
```

Comment:
We should make sure this sticks around if we're deprecating it, otherwise folks who are using it will have failing sass builds 👀